shared_infra/dynamo_write_heartbeat/src/test_dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,56 @@
+# -*- encoding: utf-8 -*-
+
+import pytest
+import os
+import dynamo_write_heartbeat
+
+
+TABLE_NAMES = ['Test1', 'Test2']
+TABLE_NAME_STR = ','.join(TABLE_NAMES)
+
+
+@pytest.yield_fixture(autouse=True)
+def run_around_tests(dynamodb_client):
+    table_names = TABLE_NAMES
+
+    for table_name in table_names:
+        create_table(dynamodb_client, table_name)
+    yield
+    for table_name in table_names:
+        dynamodb_client.delete_table(TableName=table_name)
+
+
+def test_executing_lambda_no_params(dynamodb_client):
+    with pytest.raises(RuntimeError) as e:
+        dynamo_write_heartbeat.main(event=None, context=None, dynamodb_client=dynamodb_client)
+    e.match('TABLE_NAMES not found')
+
+
+def test_executing_lambda(dynamodb_client):
+    os.environ.update({'TABLE_NAMES': TABLE_NAME_STR})
+    dynamo_write_heartbeat.main(event=None, context=None, dynamodb_client=dynamodb_client)
```

Comment:
Maybe this test should include some other table items, and check they’re not affected? That feels like the most likely failure mode for this Lambda.